### DISCUSSION TITLE
Allowed a Windows 11 update - Start Menu reverted to default

### DISCUSSION THREAD

**Discussion by Major-PITA** [valinet/ExplorerPatcher#3280] [Q&A] [answered]
I never allow updates, block them with passion through Winearo, even have custom Comodo firewall and app rules to block them. Somehow one or more got through and after blindly restarting my computer the Start menu reset to default.

Uninstalled/installed Explorer Patcher 3 times, restarted File Explorer, and everything I set through Explorer Patcher 'sticks' except the Start Menu. Even the Start button is changed.

What am I missing, what else can I do?

Just wanted to add, my Windows 11 Pro build shows as v 22H2 build 22621.3593

**Comment by Amrsatrio** [comment, marked as answer]
What's your EP version? The latest security update crippled the 10 start menu further requiring a new method to enable.

Install this if you haven't:
https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.1_0976666

**Reply by Major-PITA** [reply]
That's the cow's milk. THANK YOU!

FWIW I just threw some dollars at this project.

**Reply by IndySpirit** [reply]
Amrsatrio I'm having this issue as well and for the same reason.  However, if I understand the description of the EP release 65.2 correctly, if I update to it from my version (22621.3296.64.1), I will lose the ability to continue using the Windows 10 Taskbar - is that correct?
Am I stuck with the Windows 11 Taskbar if I want my Start menu settings back?

**Reply by Amrsatrio** [reply]
@IndySpirit You can update your EP without issues if you are running 22621 or 22631 (22H2 or 23H2).

**Reply by IndySpirit** [reply]
Awesome thank you!  I _am_ running 22631, so I applied EP release 65.2 and it fixed the issue, and _does_ still allow the Windows 10 Taskbar.  These two are so important to me!  Thanks again.
